docs/about/changelog.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+## [X.X.X] - YYYY-MM-DD
+
+The Risk Data Library Standard adheres to [Semantic Versioning](https://semver.org/).
```

Comment:
To my knowledge, semantic versioning isn't yet implemented so I suggest that we omit this for now.

```suggestion
```